server/index.js

```diff
@@ -63,6 +62,105 @@ app.get('/api/products/:productId', (req, res, next) => {
     });
 });
 
+app.get('/api/cart/', (req, res) => {
+  const sql = `
+    SELECT "c"."cartItemId",
+            "c"."price",
+            "p"."productId",
+            "p"."image",
+            "p"."name",
+            "p"."shortDescription"
+        FROM "cartItems" AS "c"
+        JOIN "products" AS "p" USING ("productId")
+      WHERE "c"."cartId" = $1
+  `;
+  const value = [req.session.cartId];
+  db.query(sql, value)
+    .then(result => {
+      res.json(result.rows);
+    })
+    .catch(err => {
+      console.error(err);
+      return res.status(500).json({
+        error: 'An unexpected error occurred.'
+      });
+    });
```

Comment:
you can pass the `err` to `next()` here. There is an error handling middleware connected to the server (check the bottom of `index.js` that does the `console.error` and `500` for you already.

Reply:
removed them all and just added `next(err)`. Is this fine?